utils/build.js

```diff
@@ -1,49 +1,31 @@
-const fs = require('fs'),
-  ncp = require('ncp').ncp,
-  path = require('path'),
-  rimraf = require('rimraf'),
-  glob = require('glob'),
-  spawn = require('cross-spawn'),
-  lodash = require('lodash'),
-  chalk = require('chalk');
+const chalk  = require('chalk');
+const fs     = require('fs');
+const glob   = require('glob');
+const lodash = require('lodash');
+const ncp    = require('ncp').ncp;
+const path   = require('path');
+const rimraf = require('rimraf');
+const spawn  = require('cross-spawn');
 
 const TMP_FOLDER = path.join(__dirname, '..', '.tmp');
 ncp.limit = 20;
 
 module.exports = function build(dependency) {
-  const ignoredFiles = dependency.ignore.map((i) => path.join(dependency.destination, i)),
-    existingFiles = [path.join(TMP_FOLDER, dependency.name)];
+  const ignoredFiles = dependency.ignore.map(i => path.join(dependency.destination, i));
+  const existingFiles = [path.join(TMP_FOLDER, dependency.name)];
 
-  if (!!dependency) {
+  if (! lodash.isEmpty(dependency)) {
```

Comment:
I personally don't use it as a coding convention, because the space could be a broken character or some fancy ascii thing. However, it's low-hanging fruit. Up to you